03-rnaseq/pathway-analysis_rnaseq_02_gsea.Rmd

```diff
@@ -187,17 +187,44 @@ First we will assign the URL to its own variable called, `dge_url`.
 dge_url <- "https://refinebio-examples.s3.us-east-2.amazonaws.com/03-rnaseq/results/SRP123625/SRP123625_differential_expression_results.tsv"
 ```
```

Comment:
It won't allow me to comment a little further up but looks like the `##Import data` header on line 175 of this notebook still needs to be changed to `## Download data file`